internal/codespaces/ssh.go

```diff
@@ -88,6 +88,7 @@ func newSSHCommand(ctx context.Context, port int, dst string, cmdArgs []string)
 	}
 
 	cmd := exec.CommandContext(ctx, "ssh", cmdArgs...)
+	cmd.Env = os.Environ()
```

Comment:
Note that when `cmd.Env` isn't explicitly set, Go will use the environment of the parent process of any subprocesses. With that in mind, it isn't really clear to me what this change does?

Reply:
It appears that setting environment variables for your ssh session is done via SendEnv ssh configuration key:
> Specifies what variables from the local `environ(7)` should be sent to the server.  The server must also support it, and the server must be configured to accept these environment variables.  Note that the TERM environment variable is always sent whenever a pseudo-terminal is requested as it is required by the protocol.  […] Variables are specified by name, which may contain wildcard characters.  Multiple environment variables may be separated by whitespace or spread across multiple SendEnv directives.
It also looks like TERM will automatically be send. Could you tell us more about your problem?

Reply:
Oh I didn't realise that!

Sorry I left this as a draft because I hadn't had time to test the fix.

It turns out I have actually aliased the SSH command so it works nicely with alacrity terminal. 

The env want an issue at all. Sorry I'll close this one out.

Reply:
https://github.com/lawrencegripper/dotfiles/blob/ab184a63283d7ecdd4bfc9d812d1974551da736d/terminal/aliases.zsh#L19